Solutions/SonraiSecurity/Workbooks/Images/Logo/Sonrai.svg

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Size of the logo should be less than 5Kb.Please refer the [guidelines](https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/ReadMe.md)